homeassistant/components/airtouch5/cover.py

```diff
@@ -0,0 +1,139 @@
+"""Representation of the Damper for AirTouch 5 Devices."""
+import logging
+from typing import Any
+
+from airtouch5py.airtouch5_simple_client import Airtouch5SimpleClient
+from airtouch5py.packets.zone_control import (
+    ZoneControlZone,
+    ZoneSettingPower,
+    ZoneSettingValue,
+)
+from airtouch5py.packets.zone_name import ZoneName
+from airtouch5py.packets.zone_status import ZoneStatusZone
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from . import Airtouch5ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import Airtouch5Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: Airtouch5ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Airtouch 5 Cover entities."""
+    client = config_entry.runtime_data
+
+    entities: list[CoverEntity] = []
+
+    # Each zone has a cover for its open percentage
+    for zone in client.zones:
+        entities.append(
+            Airtouch5ZoneOpenPercentage(
+                client, zone, client.latest_zone_status[zone.zone_number].has_sensor
+            )
+        )
+
+    async_add_entities(entities)
+
+
+class Airtouch5ZoneOpenPercentage(CoverEntity, Airtouch5Entity):
+    """How open the damper is in each zone."""
+
+    _attr_device_class = CoverDeviceClass.DAMPER
+
+    def __init__(
+        self, client: Airtouch5SimpleClient, name: ZoneName, has_sensor: bool
+    ) -> None:
+        """Initialise the Cover Entity."""
+        super().__init__(client)
+        self._name = name
+
+        self._attr_unique_id = f"zone_{name.zone_number}_open_percentage"
+        self._attr_name = "Damper"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, f"zone_{name.zone_number}")},
+            name=name.zone_name,
+            manufacturer="Polyaire",
+            model="AirTouch 5",
+        )
+
+        # Zones with temperature sensors shouldn't be manually controlled
```

Comment:
It is okay (although not recommended) to manually control the damper for zones that have temperature sensors.
I think we remove this test and always allow it, we should update the documentation to say this isn't recommended because the AirTouch5 unit should be controlling them (assuming you have AT5 temperature sensors, which you really should have!)

Reply:
I'm happy to do that. I haven't made the doco PR yet but it's on my list of things to do.